index.html

```diff
@@ -58,15 +58,16 @@ <h3 class="total-price-title">total price:</h3>
 
     <div id="payment-modal" class="modal payment-modal hide">
         <form id="payment-form" class="payment-form">
+            <button type="button" id="close-modal-btn" class="close-btn">X</button>
             <h3 class="modal-title">Enter card details</h3>
 
-            <input id="name" class="modal-input" type="text" placeholder="Enter your name">
+            <input id="name" class="modal-input" type="text" placeholder="Enter your name" required>
 
             <input id="ccn" class="modal-input" type="tel" inputmode="numeric" pattern="[0-9\s]{13,19}"
-                autocomplete="cc-number" maxlength="19" placeholder="Enter card number">
+                autocomplete="cc-number" maxlength="19" placeholder="Enter card number" required>
 
-            <input id="cvv" class="modal-input" type="text" maxlength="3" placeholder="Enter your CVV">
-            <button id="pay-btn" class="pay-btn">Pay</button>
+            <input id="cvv" class="modal-input" type="text" maxlength="3" placeholder="Enter your CVV" required>
+            <button type="submit" vid="pay-btn" class="pay-btn">Pay</button>
```

Comment:
This line has "vid" written instead of "id".

Reply:
fixed